go.mod

```diff
@@ -0,0 +1,37 @@
+module github.com/lytics/grid
+
+go 1.12
+
+require (
+	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 // indirect
+	github.com/coreos/go-semver v0.3.0 // indirect
+	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f // indirect
+	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
+	github.com/dustin/go-humanize v1.0.0 // indirect
+	github.com/gogo/protobuf v1.3.0 // indirect
+	github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6 // indirect
+	github.com/golang/protobuf v1.3.2
```

Comment:
This doesn't match our current version (revision 4c88cc3f1a34ffade77b79abc53335d1e511f25b). I'm fine with an upgrade as long as we re-generate protobuf.